.github/workflows/test.yml

```diff
@@ -0,0 +1,138 @@
+name: Test the current PR
+on:
+  repository_dispatch:
+    types:
+      - test-command
+  push:
+    tags:
+      - 'v*'
```

Comment:
It is probably useless to run those on release, since this workflow is no longer blocking and they were already executed on master.